AbletonMCP_Remote_Script/handlers/clips.py

```
@@ -0,0 +1,730 @@
+        raise
+
+
+# --- Warp Markers ---
+
+
+def get_warp_markers(song, track_index, clip_index, ctrl=None):
+    """Get the warp markers of an audio clip."""
+    try:
+        _, clip = get_clip(song, track_index, clip_index)
+        if not clip.is_audio_clip:
+            raise ValueError("Warp markers are only available on audio clips")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)